### DISCUSSION TITLE
Fast API RedirectResponse to Azure Blob Storage URL is blocked by CORS

### DISCUSSION THREAD

**Discussion by AdelinaK** [fastapi/fastapi#6296] [Questions]
### First Check

- [ ] I added a very descriptive title to this issue.
- [x] I used the GitHub search to find a similar issue and didn't find it.
- [x] I searched the FastAPI documentation, with the integrated search.
- [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [ ] I commit to help with one of those options 👆

### Example Code

```python
def get_application():
    """Function to initialze and return the app"""
    app = FastAPI(title=settings.server_settings.title,
                  openapi_url="/openapi.json")

    app.add_middleware(
        CORSMiddleware,
        allow_origins=["*"],
        allow_credentials=True,
        allow_methods=["*"],
        allow_headers=["*"],
    )
```

### Description

I have a Fast API application and a GET request with a file id. In backend an Azure Blob URL for the file is generated. I want the user to be redirected to that URL (to download the file) when the GET endpoint is accessed. The query works as expected (in the backend I have the URL correctly generated and if I open it I can download the file) but the response is blocked, not sure if this is because of the browser or Fast API/Swagger UI limitation. I used a StreamingResponse with 200 status code before, instead of RedirectResponse and it worked. No ideea why the CORS is blocking the redirect. 
![image](https://user-images.githubusercontent.com/65292608/187425883-aad4f264-80b8-45bb-91ca-288db4ea3606.png)
![image](https://user-images.githubusercontent.com/65292608/187425996-ef80f378-2e09-454c-ae1a-cfd53f7d15b1.png)

The cors configuration is simple.

### Operating System

Windows

### FastAPI Version

0.70.1

### Python Version

3.7.0

**Comment by iudeen** [comment]
Your host is `localhost`. It will be blocked by Azure because of their policy. Since origin is not changed, this is expected behavior.
